content/en/docs/reference/config/analysis/ist0118/index.md

```diff
@@ -0,0 +1,44 @@
+---
+title: PortNameIsNotUnderNamingConvention
+layout: analysis-message
+---
+
+This message occurs when the port doesn't follow the [Istio service port naming convention](/docs/ops/deployment/requirements/)
+or the port is unnamed. If you are not on the way to hardening your cluster, you can ignore the message.
```

Comment:
Not the grammar person, but `If you are not on the way to hardening your cluster` doesn't seem normal to me. Also, would people who aren't trying to harden their clusters possibly still care about this message?

Reply:
Ok, I am going to remote that statement, WDYT?